libs/core/src/lib/select/commons/select-commons.ts

```diff
@@ -0,0 +1,820 @@
+/**
+* The MIT License
+* Format: Copyright (c) 2010-2020 Google LLC. http://angular.io/license
+* Upstream-Name: Angular material
+* Upstream-Contact: fundamental@sap.com
```

Comment:
I think this should be google, not us

Reply:
Hi @mikerodonnell89 , "google". hmm...there is no contact details shared in MIT licence from angular team.Shall I keep it blank??